.github/workflows/upgrade.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+set -eux
+
+fetch_latest(){
+  # Fetch latest version from Github API
+  #  NOTE: It uses the index number 2 instead of 0 because on moby
+  #  there are the tags xdocs-v1.10-28-mar-2016 and xdocs-v1.10-09-05-2016
+  #  being listed first
+ LATEST=$(curl -s "https://api.github.com/repos/moby/moby/tags" | jq -r '.[2].name')
```

Comment:
This will be very fragile. What if a third such tag gets added?
I suggest relying on the most recent tag using git or [release](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28) instead.